data/json/items/comestibles.json

```diff
@@ -2426,8 +2426,8 @@
         "description" : "This is a serving of crisply brined and canned fish.  Tasty and nutritious.",
         "price" : 250, "//" : "Two charges per jar.",
         "material" : "flesh",
-        "volume" : 2,
-        "charges" : 2,
+        "volume" : 1,
```

Comment:
Due to how its a flexible, folded, or otherwise hard to carry item (try carrying slick, briny fish in your hands) theres not a whole lot of reason why 2 helipings of fish is 1 volume